socorro/processor/cache_manager.py

```diff
@@ -0,0 +1,378 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This was copied over from Eliot where it's been in production for a year. Then I updated it to work in Socorro.